blocks/header/header.css

```diff
@@ -272,6 +278,196 @@
   color: inherit;
 }
 
+/* Login dialog */
+.login-dialog {
+  display: none; 
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgb(0 0 0 / 70%);
+  z-index: 1000;
+}
+
+.login-dialog .login-dialog-content {
+  font-family: var(--font-gotham);
+  width: 437px;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  background-color: #F6F6F6;
+  box-shadow: 0 0 10px rgba(0 0 0 / 50%);
+  color: #5B606B;
+  font-style: normal;
+  font-weight: 350;
+  font-size: 12px;
+  letter-spacing: 0; 
+  text-transform: none;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 25px 30px;
+}
+
+.login-dialog-close {
+  position: absolute;
+  right: 15px;
+  top: 15px;
+  font-size: 20px;
+  cursor: pointer;
+}
+
+.login-dialog-logo {
+  display: block;
+  width: 220px;
+  margin: 0 auto;
+  height: 100%;
+  padding-bottom: 18px;
+}
+
+.login-dialog h2 {
+  font-family: var(--font-tungsten);
+  font-size: 35px;
+  font-style: semibold;
+  font-weight: 600;
+  letter-spacing: 0.05em;
+  margin: 15px auto 0;
+  color: #151C2C;
+  line-height: normal;
+  padding-bottom: 25px;
+}
+
+.login-dialog-form input[type="email"],
+.login-dialog-form input[type="password"] {
+  width: 100%;
+  padding: 10px;
+  margin: 10px 0;
+  font-size: 14px;
+  font-weight: normal;
+  border: 1px solid #E7E7E7;
+  background-color: #fff;
+  height: 66px;
+}
+
+.login-dialog-form input[type="email"] {
+  margin-bottom: 30px;
+}
+
+.login-dialog .login-dialog-form label {
+  font-weight: 500;
+}
+
+.login-dialog-button-container {
+  display: flex;
+  justify-content: center;
+  width: 100%;
+  margin-top: 14px;
+  padding: 5px;
+}
+
+.login-dialog-forgot {
+  text-align: right;
+  margin: 10px;
+}
+
+.login-dialog .login-dialog-signup {
+  text-align: center;
+  margin: 0 20px 20px;
+}
+
+.login-dialog-signup .red {
+  color: #FF003A;
+  font-weight: bold;
+  margin-bottom: 35px;
+}
+
+.login-dialog .login-dialog-signup a:hover span.red {
+  color: black;
+  text-decoration: underline;
+}
+
+.login-dialog .gd-agreement-checkbox {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin-top: 10px;
+}
+
+.login-dialog .gd-agreement-checkbox input {
+  margin: 0;
+  width: 20px;
+  height: 20px;
+}
+
+.login-dialog .gd-agreement-checkbox label {
+  line-height: 2.5em;
+  width: 250px;
+  font-size: 10px;
+}
+
+.login-dialog .login-dialog-signup a {
+  color: #5B606B;
+  font-size: 14px;
+  font-weight: 500;
+}
+
+.login-dialog-signup a:hover {
+  color: black;
+  text-decoration: underline;  
+}
+
+.login-dialog .login-dialog-agreement a {
+  text-decoration: underline;
+  color: black;
+}
+
+.login-dialog .gd-agreement-checkbox label a {
+  font-size: 15px;
+}
+
+.login-dialog-agreement {
+  text-align: center;
+  margin-top: 20px;
+  font-weight: 500;
+  font-size: 10px;
+  color: #5B606B;
+}
+
+.login-dialog.open {
+  display: block;
+}
+
+.required {
```

Comment:
Do you think this can be used across blocks ? If yes then you can move this into styles.css and if not then I'd recommend adding a prefix to the selector to increase the specificity

Reply:
I take a look!